src/routes/auth.js

```diff
@@ -22,7 +22,7 @@ const router = Router();
 router.post(
   '/register',
   UserValidation.verifyUser,
-  EmailValidation.checkEmail,
+  // EmailValidation.checkEmail,
```

Comment:
Remove this

Reply:
This is the middleware for checking valid email, it is commented because the deep email validator has issue. If resolved that middleware will be uncommented.